server/routes/webmaker-app.js

```diff
@@ -16,11 +17,27 @@ var webmakerVersion = require('../../package.json').dependencies.webmaker;
 if (!habitat.get('FIREBASE_ENDPOINT')) throw new Error('You must configure FIREBASE_ENDPOINT in .env');
 
 var firebase = new Firebase(habitat.get('FIREBASE_ENDPOINT') + '/apps');
+var ircBot = habitat.get('BOT_ENDPOINT');
 
 module.exports = function (req, res, next) {
     var user = req.session && req.session.user;
     var baseDir = 'p';
 
+    function tellIRC (evt, data) {
```

Comment:
You need to do a check somewhere if `ircBot` is truthy; you can either early return in this function, or do a conditional check where you run `tellIRC`